css/style.css

```diff
@@ -163,11 +174,27 @@ header {
 }
 
 .banner-text {
+
   position: absolute;
   top: 35%;
   z-index: 1;
   margin-left: 6rem;
   margin-right: 6rem;
+
+}
+
+.banner-text .main-head {
+  color: #fff;
+  font-size: 4rem;
+  margin-left: 2vw;
+    position: absolute;
```

Comment:
Correct indentations and check the styles in different resolutions as I can see that the typing text is being overlapped.